packages/vats/src/lib-board.js

```diff
@@ -32,40 +31,50 @@ const calcCrc = num => {
 /**
  * Create a board to post things on.
  *
- * @param {number} [seed=0]
+ * @param {number | bigint} [firstSequence=0]
+ * @param {string} [prefix]
  * @returns {Board}
  */
-function makeBoard(seed = 0) {
+function makeBoard(firstSequence = 0, prefix = 'board0') {
   const idToVal = makeStore('boardId');
   const valToId = makeStore('value');
-  const sparseInts = generateSparseInts(seed);
+  let lastSequence = BigInt(firstSequence);
 
   /** @type {Board} */
   const board = Far('Board', {
     // Add if not already present
     getId: value => {
       if (!valToId.has(value)) {
-        // Retry until we have a unique id.
-        let id;
-        do {
-          const num = /** @type {number} */ (sparseInts.next().value);
-          id = `${num}${calcCrc(num)}`;
-        } while (idToVal.has(id));
+        lastSequence += 1n;
+        const seq = `${lastSequence}`;
+
+        // Reverse the sequence, so that the initial part of the board ID is
+        // well-distributed.
```

Comment:
Why is it important for apparent number to be well-distributed? You don't want the ordering to be so obvious?

Reply:
I find that people when comparing two large numbers tend to look at the first few digits and the last few digits.  Let me address this below....